.changeset/neat-walls-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Add config `processors` that allows modifying lint results.
```

Comment:
[suggest] Let's adjust the format for consistency. Ref https://stylelint.io/maintainer-guide/pull-requests#merging

```suggestion
Added: experimental support for post processors
```